package.json

```diff
@@ -9,16 +9,17 @@
   "license": "MIT",
   "main": "dist/app.js",
   "scripts": {
-    "clean": "rm -rf dist",
+    "clean": "rimraf dist",
     "build": "yarn run clean && webpack-dashboard -- webpack --config configs/webpack.config.js --env.build --env.sourceMap",
     "dev": "webpack-dashboard -m -- webpack-dev-server --config configs/webpack.config.js --open",
     "dev2": "webpack-dashboard -- webpack-dev-server --config configs/webpack.config.js --open --env.awesome",
-    "lint": "tslint --project 'tsconfig.json'",
+    "lint": "tslint --project \"tsconfig.json\"",
     "tsc": "tsc -p . --noEmit",
     "tsc:watch": "tsc -p . --noEmit -w",
     "test": "jest --config jest.json",
     "test:update": "jest --config jest.json -u",
     "re": "rm -rf node_modules && yarn",
+    "rewin": "rd /s /q node_modules && yarn",
```

Comment:
ah, that was my first try. Unfortunately that doesn't work, because rimraf is in node_modules, and Windows won't let you delete it while it's being used. This is a known problem with rimraf on Windows: https://github.com/isaacs/rimraf/issues/102
One alternative could be to attempt to remove everything _but_ rimraf, but I had difficulties getting that to work... Happy to give it another shot if you reckon it's worth the effort